### DISCUSSION TITLE
How to move search bar back to sidenav in Sphinx Book Theme v1.0.0

### DISCUSSION THREAD

**Discussion by ketozhang** [executablebooks/meta#959] [Q&A]
I don't really like the aesthetic of having the search bar within the main body.  It looks like you're searching within the body and not throughout the website.

How do I move it back to the way it was before v1.0.0. I understand this might be tied to the pydata theme version.

**Comment by haesleinhuepf** [comment]
I also would like to know how to get the search bar back into the side navigation on top of the TOC. It was very intuitive to search from there. @choldgraf can you maybe assist?

Thanks!

**Comment by choldgraf** [comment]
I'm actually wondering if we should move it back to the sidenav by default. We inherited the icon from the pydata theme, and while I like the search popup, I agree that the sidenav was a more intuitive place for it. In my mind, the page icons are for page-specific buttons while the sidenav is for cross-site content.

Do others agree with this take? (I suppose @haesleinhuepf and @ketozhang do)

**Reply by ketozhang** [reply]
Absolutely!

**Comment by iimog** [comment]
It looks like the search bar is back in the sidenav in the current version (1.1.3). I like it.
